main/src/main/scala/sbt/Load.scala

```diff
@@ -585,7 +585,9 @@ object Load {
               val existingIds = otherProjects.projects map (_.id)
               val refs = existingIds map (id => ProjectRef(buildUri, id))
               val defaultID = autoID(buildBase, context, existingIds)
-              val root = finalizeProject(Build.defaultAggregatedProject(defaultID, buildBase, refs), files)
+              val root0 = if (discovered.isEmpty || java.lang.Boolean.getBoolean("sbt.root.ivyplugin")) Build.defaultAggregatedProject(defaultID, buildBase, refs)
```

Comment:
This doesn't look right. getBoolean on the raw string instead of the actual property variable....